API_doc/Communication.md

```diff
@@ -15,15 +17,28 @@ Each response contain at least **REQUEST_STATE** (see below).
 |*SETTING_GET*| Get setting |**CONFIG_ID**<br>**SETTING_NAME**|**SETTING_VALUE**|
 |*ALIAS_SET*| Update or create alias |**CONFIG_ID**<br>**SETTING_NAME**<br>**ALIAS_NAME**|*none*|
 |*ALIAS_UNSET*| Unset alias |**CONFIG_ID**<br>**ALIAS_NAME**|*none*|
+|*SUBSCRIBE_SETTING*| Subscribe to given setting |**CONFIG_ID**<br>**SETTING_NAME** *or* **ALIAS_NAME**|*none*|
+|*UNSUBSCRIBE_SETTING*| Unsubscribe to given setting |**CONFIG_ID**<br>**SETTING_NAME** *or* **ALIAS_NAME**|*none*|
 
 ### REQUEST_STATE
 
-
 | Value | Meaning |
 | -------- | -------- |
 | SUCCESS | Everything ok |
 | BAD_ORDER | Argument(s) missing or bad |
 | UNKNOWN_REQUEST | Bad request name |
 | INTERNAL_ERROR | The service got an error |
-| UNKNOWN_ID | Given config id don't exist|
-| UNKNOWN_KEY | Given config key don't exist|
+| UNKNOWN_ID | Given config id doesn't exist|
+| UNKNOWN_KEY | Given config key doesn't exist|
+| UNKNOWN_SETTING | Given setting name doesn't exist|
+| UNKNOWN_ALIAS | Given alias name doesn't exist|
+
+## Events
+
+The service can also send a message to the lib if a subscribed setting is modified.
+
+| Argument name | Description |
+| ---- | ---- |
+| *SETTING_NAME* | Name of the concerned setting. |
```

Comment:
Either copy this into both other actions, or explain in preamble that this parameter is always present.